x-pack/legacy/plugins/siem/public/mock/kibana_react.ts

```diff
@@ -71,7 +71,18 @@ export const createUseUiSetting$Mock = () => {
 };
 
 export const createUseKibanaMock = () => {
-  const services = { ...createKibanaCoreStartMock(), ...createKibanaPluginsStartMock() };
+  const core = createKibanaCoreStartMock();
+  const plugins = createKibanaPluginsStartMock();
+  const useUiSetting = createUseUiSettingMock();
+
+  const services = {
+    ...core,
+    ...plugins,
+    uiSettings: {
+      ...core.uiSettings,
+      get: useUiSetting,
```

Comment:
If we wanted to be really thorough, we could have `get$: createUseUiSetting$mock()` here as well, but I don't believe we're using that method anywhere right now (opting instead for the `useUiSetting$` hook).